types/es-abstract/2015/SameValueZero.d.ts

```diff
@@ -0,0 +1,2 @@
+declare function SameValueZero(x: unknown, y: unknown): boolean;
```

Comment:
can this be a predicate that asserts that `x` and `y` are the same type?

Reply:
Unfortunately, that’s not easy to do.